client/components/table/style.scss

```diff
@@ -50,6 +51,16 @@
 	}
 }
 
+.woocommerce-table__shadow {
+	content: '';
```

Comment:
I think this can be removed.
```suggestion
```

Reply:
You're right; this was left from a previous implementation and should not have made it into the PR 😓 - Removed!